src/Directory.Build.props

```diff
@@ -19,6 +19,7 @@
       <!-- Disable code analysis for source build -->
       <PropertyGroup>
         <EnableNETAnalyzers>false</EnableNETAnalyzers>
+        <RunAnalyzers>false</RunAnalyzers>
       </PropertyGroup>
```

Comment:
We only include the following in non-source-build.

```xml
        <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="$(MicrosoftCodeAnalysisAnalyzersVersion)" />
```

So in source-build, we are supposed to not get the package. However, we get it as transitive dependency, and the transitive dependency is 3.0.0 which has the bug, causing source-build to fail.

Per `<!-- Disable code analysis for source build -->`, it seems we don't want to run analyzers for source-build. So I added `<RunAnalyzers>false</RunAnalyzers>`